loopchain/baseservice/timer_service.py

```diff
@@ -44,6 +43,7 @@ def __init__(self, **kwargs):
         self.__duration = kwargs.get("duration")
         self.__start_time = time.time()
         self.__is_repeat = kwargs.get("is_repeat", False)
+        self.is_run_at_start = kwargs.get("is_run_at_start", False)
```

Comment:
Why don't you is_run_at_start move to add_timer function parameter?
is_run_at_start just need at first time(add_timer function)